Sources/FluidStack/Helper/FluidExtentionViewController.swift

```diff
@@ -209,7 +209,6 @@ extension FluidExtentionViewController {
    - Parameters:
    - transition: You may set ``AnyRemovingTransition/noAnimation`` to disable animation, nil runs transition given view controller provides (if it's ``FluidTransitionViewController``).
    - fowardingToParent: Forwards to parent to pop if current stack do not have view controller to pop. No effects if the current stack prevents it by ``FluidStackController/Configuration-swift.struct/preventsFowardingPop``
-   - cascadesToChildren: if true and if there are view controllers stacked above this view controller, they will be removed as well
```

Comment:
just removed this line because `removingRule` is explained in `enum RemovingRule`